### ISSUE TITLE
Must supply apiKey in Laravel 11

### ISSUE THREAD

**Issue by hamidroohani** [cloudinary-labs/cloudinary-laravel#120]
I recently added this package to my laravel project which is 11, PHP 8.3

```
->saveUploadedFileUsing(function ($file, $state) {

         return Cloudinary::upload($file->getRealPath())->getSecurePath();
         
 })
```
 

 
```
 CLOUDINARY_URL=XXXXXXXX
 
CLOUDINARY_API_KEY=public_XXXXX

CLOUDINARY_API_SECRET=private_XXXXXX

CLOUDINARY_CLOUD_NAME=xcvmshtrd
```

I tried to clear cache but nothing changed and always I get the same error
![Screenshot 2024-07-30 at 11 51 17](https://github.com/user-attachments/assets/eb93ada7-e3dd-44d1-b843-60f1f2d7f7fe)

**Comment by joshmanders** [comment]
Hey @hamidroohani we're working on fixing the package and getting it working in Laravel 10 and 11, there's a few issues we're working through right now, I apologize for any issues you're running into.

**Comment by hamidroohani** [comment]
I asked Chatgbt 3 times "suggest me a laravel package for Cloudinary" and it always suggests this package. so it's worth it
Thanks

**Comment by hamidroohani** [comment]
and also another error when I try to use it as a disk in filament

```
 FileUpload::make('logo')
      ->image()
      ->disk('cloudinary')
```

![Screenshot 2024-07-31 at 15 46 53](https://github.com/user-attachments/assets/7f4f0559-3987-4565-baf5-373a73d05a61)

**Comment by joshmanders** [comment]
Hey @hamidroohani I published v2.2.0 yesterday which should make it work in Laravel 10 and 11. Go ahead and give it a try and if you see any problems, please open a new issue so I can address it.
